src/Bicep.Core/Bicep.Core.csproj

```diff
@@ -23,6 +23,7 @@
 		<PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
 		<PackageReference Include="Azure.Deployments.Expression" Version="1.0.173" />
 		<PackageReference Include="Azure.Deployments.Core" Version="1.0.173" />
+    	<PackageReference Include="Azure.Deployments.Templates" Version="1.0.173" />
```

Comment:
Thinking about this a little more - I think it would be worth restructuring our dependencies in the deployments repo. Unfortunately `Azure.Deployments.Templates` depends on `Azure.Deployments.TemplateSchemas`, which is a huge DLL containing lots of JSON; I'd like to avoid this increasing the size of the binaries.

Reply:
build from main:
![image](https://user-images.githubusercontent.com/38542602/121073473-909f8400-c7a0-11eb-80bb-bb69a685998a.png)

this build:
![image](https://user-images.githubusercontent.com/38542602/121073623-bc226e80-c7a0-11eb-8f52-5ac015401db1.png)

Reply:
@TarunSunkaraneni I've made this change - available in [1.0.207](https://www.nuget.org/packages/Azure.Deployments.TemplateSchemas/1.0.207). Please could you bump the package version in this PR?

Reply:
What i got after the change. Thanks @anthony-c-martin! 
![image](https://user-images.githubusercontent.com/3432839/121726535-4b38ca80-cab0-11eb-8c67-fb6ba0039750.png)